README.md

```diff
@@ -23,7 +26,7 @@ $ npm install -g eb-scripts
 $ eb-scripts COMMAND
 running command...
 $ eb-scripts (-v|--version|version)
-eb-scripts/0.0.0 darwin-x64 node-v10.16.0
+eb-scripts/0.0.0-development darwin-x64 node-v10.16.0
```

Comment:
why does this keep coming back?

Reply:
`oclif` makes this harder than it should be.

When we triggered the first release, `semantic-release` created a `v1.0.0` but it did not update the version in the `package.json`. As a result, `oclif` has left the `0.0.0-development` as the version.

My tests are failing because I'm getting this warning:
```
Warning: Mismatched version in eb-scripts plugin manifest. Expected: 1.0.0 Received: 0.0.0
```

I thought if I pulled in the original `oclif.manifest.json` from `master`, that that would fix it. But it didn't, so now I need to figure out a different approach. I may have to manually update the `package.json` in `master` and the `oclif.manifest.json`, then merge into `next` and go from there.

Reply:
🤔 I wonder if this is b/c we didn't have the symantic-release plugins initially. Perhaps we can sync the `package.json` manually like you mention and be in good shape next time around?

Reply:
> I wonder if this is b/c we didn't have the symantic-release plugins initially.

Yup! This is exactly why. 

> we can sync the package.json manually like you mention and be in good shape next time around?

I went ahead and did that, so 🤞 it works.